### ISSUE TITLE
June 2024 Goals

### ISSUE THREAD

**Issue by lola3736** [hackforla/HomeUniteUs#714]
By 6/30/2024

#### In progress:
- [ ] #440 - PM and Design to discuss today
    - [x] Pending acknowledgement functionality - Prototype for user testing in progress, some complexity noted
- [ ] Analysis and study anticipated to complete June 30th 
   - Study has launched, still need 2 more participants, but on track
- [ ] Guest application: User testing. Research plan

#### New Items:
- [ ] Coordinator notes: Save and cancel + editing screens for notes #519 
- [ ] Coordinator: Assign and reassign applications #518
- [ ] Host Dashboard #508 
- [ ] Host Application #509 
- [ ] Guest/Host sign up: Add acknowledgement functionality
- [ ] Move guest minimum expectations to sign up process
- [ ] Coordinator dashboard updates #501: Add status prior to submission. Add a column to have link to application
- [ ] Bonnie would like to see what can be done for Guest UI/UX to feel fun and positive

#### In progress:
- [ ] Account Creation
  - [ ] #703 - manual testing - up for review, needs to approved
- [ ] Guest Intake Profile | #708
    - [ ]  #710 - Erik still working on this
    - [x] #707 - merged and complete
    - [x] #712 - merged and complete
    - [ ] Guest form endpoints #636
      - [x] Josh to complete and roll off project
      - [ ] PR needs to be reviewed
      - [ ] data models will still need to be updated, and other planning, to be handled on separate PR
- [ ] Incubator: deploy lower envs with end-to-end smoke test for SU/SI flows (i.e. existing functionality as of Mar 2024) #686
   - [ ] Anticipated migration May 31st (original completion date April 30th)
   - [x] PR merged; credential accounts created to run deployment
   - [x] Access granted to administer share AWS environment
   - [x] Major dependency - Cognito to support user management
   - [ ] terraform deploying tonight (6/25), will have support tomorrow from ops meeting
   - [ ] Bonnie will need to update the record at the name server for homeunite.us after deployment
   - [ ] smoke test
- [ ] Guest dashboard #572 #566
     - No one is currently working on #572
- [ ] Josh being rolled off the project 
- [ ] We will Need to scope first, but might need another backend dev
- [ ] Coordinator invite guest #520
   - [ ] Engineering to review acceptance criteria to finalize (some functionality may be more appropriate for new stories, but others may be valid and not yet present in dev.homeunite.us)

#### New Items:
- [ ] Coordinator dashboard endpoint #651 
- [ ] Add acknowledgement during signup
- [ ] Guest intake profile mobile 

#### Final reviews:
- [ ] Host Matching Profile view 
- [ ] Guest/host sign up: add acknowledgement functionality #439

#### In Progress
- [ ] Host dashboard - bring to parity with guest + add steps for home inspection + to modify application after training
   - [ ] Ready to Review
- [ ] Section 2 Coordinator User stories - need review, starting from Epic #454 

#### New:
- [ ] Host application -  bring to parity with guest + add a way to upload house or renters insurance
   - [ ] Update host dashboard to match guest (coordinator doesn't get assigned until after they submit their application)
- [ ] FAQ user story
- [ ] Coordinator update meeting & interview attendance/status for guest/host 
- [ ] Guest matching profile
- [ ] Create Guest, Host and Coordinator Sidebar stories
- [ ] Host experience for matching
- [ ] Downloadable intake profile instead of email
- [ ] Log of when guest and host worked on the app: V2
- [ ] Background Checks for Guest/Hosts
    - [ ] Coordinator will need a checkbox to confirm background check - update flow and add to/create a story
    - [ ] Bonnie recommends we have an additional tasks section that allows Coordinator to enter whatever tasks they need (basic text section)
    - [ ] What else does Guest need? Guest PM needs to work through this
    - [ ] What else does Host need? Host PM needs to work through this
    - [ ] What else does Coordinator need? Coordinator PM needs to work through this

**Comment by sanya301** [comment]
### Design:

#### In progress:
- [ ] #440 - completed, need to review with PMs
- [ ] Guest application mobile: Prototype for user testing in progress, some complexity noted 
        - Program they want to use for testing finalized
        - Complexity noted for being able to navigate to any section
        - @edela0015 create an issue for the research/user testing
        - [ ] Review button might not look like a button
        - [ ] Add expand collapse for Application sections in desktop
        - [ ] Submit button placement on review page
        - [ ] Text that specifies they can move around in the application without completing all the sections
- [ ] Coordinator filters: updated design for review
- [ ] Coordinator notes: Save and cancel + editing screens for notes #519 
- [ ] Coordinator: Assign and reassign applications #518
    
#### New Items:
- [ ] Host Dashboard #508 
- [ ] Host Application #509 
- [ ] Guest/Host sign up: Add acknowledgement functionality
- [ ] Move guest minimum expectations to sign up process
- [ ] Coordinator dashboard updates #501: Add status prior to submission. Add a column to have link to application

### Eng:

#### In progress:
- [ ] Account Creation
  - [ ] #703 - manual testing
- [ ] Guest Intake Profile | #708
    - [ ]  #710
    - [ ] #707 - in review
    - [ ] #712 - in review
    - [ ] Guest form endpoints #636
      - [ ] Josh to complete and roll off project
- [ ] Incubator: deploy lower envs with end-to-end smoke test for SU/SI flows (i.e. existing functionality as of Mar 2024) #686
   - [ ] Anticipated migration May 31st (original completion date April 30th)
   - [ ] PR merged; credential accounts created to run deployment
   - [ ] Access granted to administer share AWS environment
   - [ ] Major dependency - Cognito to support user management
- [ ] Guest dashboard #572 #566
     - No one is currently working on #572

#### New Items:
- [ ] Coordinator invite guest 
- [ ] Coordinator dashboard endpoint #651 
- [ ] Add acknowledgement during signup
- [ ] Guest intake profile mobile 

### PM:

#### Final reviews:
- [ ] Coordinator User stories: Application review page, comments, notes etc
- [ ] Host Matching Profile view 
- [ ] Guest/host sign up: add acknowledgement functionality #439


#### New:
- [ ] Host dashboard - bring to parity with guest + add steps for home inspection + to modify application after training
- [ ] Host application -  bring to parity with guest + add a way to upload house or renters insurance
   - [ ] Update host dashboard to match guest (coordinator doesn't get assigned until after they submit their application)
- [ ] FAQ user story
- [ ] Coordinator update meeting & interview attendance/status for guest/host 
- [ ] Guest matching profile
- [ ] Create Guest, Host and Coordinator Sidebar stories
- [ ] Host experience for matching
- [ ] Downloadable intake profile instead of email
- [ ] Log of when guest and host worked on the app: V2

**Comment by lola3736** [comment]
### Design:

#### In progress:
- [ ] #440 - completed, need to review with PMs
        -   Pending acknowledgement functionality, Move guest minimum expectations to sign up process
- [ ] Guest application mobile: Prototype for user testing in progress, some complexity noted 
        -  add issue created. work still ongoing
        - Program they want to use for testing finalized
        - Complexity noted for being able to navigate to any section
           - Able to resolve issue with the complexity.  
           - Testing tentative end of June; need to add link of the study (limited) PM to view as team on thursday 
        - @edela0015 create an issue for the research/user testing
        - [ ] Review button might not look like a button
        - [ ] Add expand collapse for Application sections in desktop
        - [ ] Submit button placement on review page
        - [ ] Text that specifies they can move around in the application without completing all the sections
- [ ] Coordinator filters: updated design for review
    
#### New Items:
- [ ] Coordinator notes: Save and cancel + editing screens for notes #519 - need to be reviewed
- [ ] Coordinator: Assign and reassign applications #518 - need to be reviewed
- [ ] Host Dashboard #508 
- [ ] Host Application #509 
- [ ] Coordinator dashboard updates #501: Add status prior to submission. Add a column to have link to application

### Eng:

#### In progress:
- [ ] Account Creation - Need to confirm Erik (merge and test)
  - [ ] #703 - manual testing. Have it in a good place, should have a PR up soon
- [ ] Guest Intake Profile | #708 - Erick to confirm below open items
     - [ ] - @sanya301 review once 
     - have general scaffolding, render the fields, navigate between 
    - [ ] #710 - working on this
    - [x ] #707 - in review - completed and merged
    - [ ] #712 - in review
    - [ ] Guest form endpoints #636 - waiting to hear back from @Joshua-Douglas 
      - [ ] Josh to complete and roll off project
        - [ ] Double check all items assigned to Josh and ensure proper handover
- [ ] Incubator: deploy lower envs with end-to-end smoke test for SU/SI flows (i.e. existing functionality as of Mar 2024) #686
   - waiting on ops team to implement some of the terraform changes
   - Still ongoing, still having issues with the infrastructure
   - Cognito has been set up, need to add home unite us specific things 
   - Anticipated migration hopefully after above two are complete
- [ ] Guest dashboard - endpoints/data modeling #572 - @erikguntner can you re-review it
- [ ] Coordinator invite guest #520  - mostly done, need to be confirmed by @lasryariel  that this is good to close

#### New Items:
- [ ] Coordinator dashboard endpoint #651 - waiting on dashboard design
- [ ] Add acknowledgement during signup - waiting on design

### PM:

#### Final reviews:
- [x] Migration of project board
- [ ] Coordinator User stories: Application review page, comments, notes etc
     - Working on clarify epic #454  move to new
     - Clarify user filter scenario for #501  move to new
     - Confirming user story for "invite Guest" #520 move new; few items in acceptance criteria that haven't been created move to new
- [ ] Host Matching Profile view - PM (sanya) to review #617   
- [ ] Guest/host sign up: add acknowledgement functionality #440 
- [ ] Project hygiene

#### New:
- [ ] Host dashboard - bring to parity with guest + add steps for home inspection + to modify application after training
- [ ] Host application -  bring to parity with guest + add a way to upload house or renters insurance
   - [ ] Update host dashboard to match guest (coordinator doesn't get assigned until after they submit their application)
- [ ] FAQ user story
- [ ] Coordinator update meeting & interview attendance/status for guest/host 
- [ ] Guest matching profile
- [ ] Create Guest, Host and Coordinator Sidebar stories
- [ ] Host experience for matching
- [ ] Downloadable intake profile instead of email
- [ ] Log of when guest and host worked on the app: V2

**Comment by lasryariel** [comment]
### Design:

#### In progress:
- [ ] #440 - completed, need to review with PMs
        -   Pending acknowledgement functionality, Move guest minimum expectations to sign up process
- [ ] Guest application mobile: Prototype for user testing in progress, some complexity noted 
        -  add issue created. work still ongoing
        - Program they want to use for testing finalized
        - Complexity noted for being able to navigate to any section
           - Able to resolve issue with the complexity.  
           - Testing tentative end of June; need to add link of the study (limited) PM to view as team on thursday 
        - @edela0015 create an issue for the research/user testing
        - [ ] Review button might not look like a button
        - [ ] Add expand collapse for Application sections in desktop
        - [ ] Submit button placement on review page
        - [ ] Text that specifies they can move around in the application without completing all the sections
- [ ] Coordinator filters: updated design for review
    
#### New Items:
- [ ] Coordinator notes: Save and cancel + editing screens for notes #519 - need to be reviewed
- [ ] Coordinator: Assign and reassign applications #518 - need to be reviewed
- [ ] Host Dashboard #508 
- [ ] Host Application #509 
- [ ] Coordinator dashboard updates #501: Add status prior to submission. Add a column to have link to application

### Eng:

#### In progress:
- [ ] Account Creation - Need to confirm Erik (merge and test)
  - [ ] #703 - manual testing. Have it in a good place, should have a PR up soon
- [ ] Guest Intake Profile | #708 - Erick to confirm below open items
     - [ ] - @sanya301 review once 
     - have general scaffolding, render the fields, navigate between 
    - [ ] #710 - working on this
    - [x] #707 - in review - completed and merged
    - [ ] #712 - in review
    - [ ] Guest form endpoints #636 - waiting to hear back from @Joshua-Douglas 
      - [ ] Josh to complete and roll off project
        - [ ] Double check all items assigned to Josh and ensure proper handover
- [ ] Incubator: deploy lower envs with end-to-end smoke test for SU/SI flows (i.e. existing functionality as of Mar 2024) #686
   - waiting on ops team to implement some of the terraform changes; still having issues with the infrastructure.
   - Cognito has been set up, need to add home unite us specific things 
   - Anticipated migration hopefully after above two are complete
- [ ] Guest dashboard - endpoints/data modeling #572 - @erikguntner can you re-review it
- [ ] Coordinator invite guest #520  - mostly done, need to be confirmed by @lasryariel  that this is good to close

#### New Items:
- [ ] Coordinator dashboard endpoint #651 - waiting on dashboard design
- [ ] Add acknowledgement during signup - waiting on design

### PM:

#### In progress:
- [ ] Coordinator User stories
   - [ ] Coordinator epic (user journey) needs to be updated and presented to the design team. Define by end of this section, coordinator needs to do this @lasryariel #454 
      - [x] Initial updates made by Ariel, needs review by PM team 
   - [ ] @lasryariel update filter user story to define name filter requirements if any, and use cases for search bar.  #501 
      - [x] Initial updates made by Ariel, needs review by PM team 
   - [x] Confirm user story for "invite Guest" #520, a few items in acceptance criteria that haven't been created - There are a few remaining criteria which can only be assessed once https://dev.homeunite.us/ coordinator dashboard is updated with actual data from invited guests. 
      - [ ] Does engineering have a ticket for this?
   - [ ] All Section 2 Coordinator stories have been refreshed, needs review by PM team. 

#### Final reviews:
- [ ] Host Matching Profile view - PM (sanya) to review #617   
- [ ] Guest/host sign up: add acknowledgement functionality #440 

#### New:
- [ ] Host dashboard - bring to parity with guest + add steps for home inspection + to modify application after training
- [ ] Host application -  bring to parity with guest + add a way to upload house or renters insurance
   - [ ] Update host dashboard to match guest (coordinator doesn't get assigned until after they submit their application)
- [ ] FAQ user story
- [ ] Coordinator update meeting & interview attendance/status for guest/host 
- [ ] Guest matching profile
- [ ] Create Guest, Host and Coordinator Sidebar stories
- [ ] Host experience for matching
- [ ] Downloadable intake profile instead of email
- [ ] Log of when guest and host worked on the app: V2
